dev/tasks/utils/dev-plugin-create.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const inquiries = require( './inquiries' );
+const git = require( './git' );
+const tools = require( './tools' );
+const path = require( 'path' );
+
+/**
+ * 1. Ask for new plugin name.
+ * 2. Ask for initial version.
+ * 3. Ask for GitHub URL.
+ * 4. Initialize repository
+ * 		4.1. Initialize GIT repository.
+ * 		4.2. Fetch and merge boilerplate project.
+ * 5. Update package.json file in new plugin's repository.
+ * 6. Update package.json file in CKEditor5 repository.
+ * 7. Link new plugin.
+ * 8. Call `npm install` in plugin repository.
+ * 9. Install Git hooks in plugin repository.
+ *
+ * @param {String} ckeditor5Path Path to main CKEditor5 repository.
+ * @param {Object} options grunt options.
+ * @param {Function} writeln Function for log output.
+ * @param {Function} writeError Function of error output
+ * @returns {Promise} Returns promise fulfilled after task is done.
+ */
+module.exports = ( ckeditor5Path, options, writeln, writeError ) => {
+	return new Promise( ( resolve, reject ) => {
```

Comment:
Do you need this promise? I think that you can return that chain which starts with `inquiries.getPluginName()`.